### PR TITLE
Prevents the look of Krav maga from being changed in a washing machine

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Prevents Krav Maga security gloves from being put in a washing machine to change their colour

## Why It's Good For The Game

Disguising steal items as other things is not good, balance wise, and the only way to tell these gloves are the real thing is to put them on. Not a good thing, not a feature we want. 

## Changelog
:cl:
tweak: You can no longer put krav maga gloves in the washing machine.
/:cl:

### CODE REVIEW

File: code/game/machinery/washing_machine.dm [hunk 1]
```diff
@@ -272,6 +272,9 @@
 		if( istype(W,/obj/item/clothing/gloves/furgloves ) )
 			to_chat(user, "This item does not fit.")
 			return
+		if( istype(W,/obj/item/clothing/gloves/color/black/krav_maga/sec ) )
```
Comment (SabreML) [inline code/game/machinery/washing_machine.dm:275]: Spacing, although it *is* consistent with the rest of the file.
```suggestion
		if(istype(W, /obj/item/clothing/gloves/color/black/krav_maga/sec))
```

File: code/game/machinery/washing_machine.dm [hunk 2]
```diff
@@ -272,6 +272,9 @@
...
+			to_chat(user, "Washing these gloves would fry the electronics!")
```
Comment (SabreML) [inline code/game/machinery/washing_machine.dm:276]: Might want a span here?
```suggestion
			to_chat(user, "<span class='warning'>Washing these gloves would fry the electronics!</span>")
```

**Comment by Varghh** [comment]
....you can do that? The fuck

**Comment by DeveloperLoser** [comment]
> ....you can do that? The fuck

Yea, just pop a crayon in with it and bam, you now have "insuls" (assuming you put a yellow crayon in).

**Comment by Wojz** [comment]
oh fuck I wish I knew that was possible for fashion reasons

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**